src/sentry/web/frontend/debug/mail.py

```diff
@@ -98,11 +98,18 @@ def make_group_generator(random, project):
 
 # TODO(dcramer): use https://github.com/disqus/django-mailviews
 class MailPreview(object):
-    def __init__(self, html_template, text_template, context=None):
+    def __init__(self, html_template, text_template, context=None, subject_template=None):
         self.html_template = html_template
         self.text_template = text_template
+        self.subject_template = subject_template
         self.context = context if context is not None else {}
 
+    def subject(self):
+        if self.subject_template:
+            return render_to_string(self.subject_template, self.context)
```

Comment:
I removed a place where we did this last week (sentry@a9ef38e6f77064b4633c10c42d9070487bf25e0b) because it's easy to get the unintended autoescaping behavior. Also, many of the emails don't actually use a template to render their subject. It's probably better just to pass the subject in as a string.